apps/dsp-app/src/app/user/user-form/user-form.component.html

```diff
@@ -1,92 +1,73 @@
-<!-- content -->
-<dasch-swiss-app-progress-indicator *ngIf="loadingData"></dasch-swiss-app-progress-indicator>
-<div *ngIf="!loadingData">
-  <form [formGroup]="userForm" class="form-content user-data" (ngSubmit)="submitData()">
-    <!-- username -->
-    <mat-form-field [class.large-field]="!user" [class.small-field]="user">
-      <input
-        matInput
-        type="text"
-        formControlName="username"
-        placeholder="{{ 'appLabels.form.user.general.username' | translate }}" />
-      <mat-hint *ngIf="formErrors.username"> {{ formErrors.username }} </mat-hint>
-    </mat-form-field>
+<form [formGroup]="userForm" class="form-content user-data">
+  <mat-form-field class="{{editExistingUser ? 'large-field' : 'small-field'}}">
+    <input
+      matInput
+      type="text"
+      formControlName="username"
+      placeholder="{{ 'appLabels.form.user.general.username' | translate }}" />
+    <mat-hint *ngIf="isInvalid('username')"> {{ getErrorMessage('username') }} </mat-hint>
+  </mat-form-field>
```

Comment:
you probably can use a app-common-input here. Look at other implementations for examples (reusable-project-form for example)